pkg/services/vmoperator/control_plane_endpoint.go

```diff
@@ -41,6 +41,14 @@ const (
 	nodeSelectorKey    = "capv.vmware.com/cluster.role"
 	roleNode           = "node"
 	roleControlPlane   = "controlplane"
+
+	// TODO(lubronzhan): Remove in the future release once deprecation process is decided.
```

Comment:
just mark it as `deprecated, will be removed in a future release`

Reply:
Done